src/mjolnir/graphfilter.cc

```diff
@@ -143,7 +143,7 @@ void FilterTiles(GraphReader& reader,
         // encoded shape plus way Id.
         bool added;
         uint32_t idx = directededge->edgeinfo_offset();
```

Comment:
this line makes me think that you should do the same with `GraphTile::GetNames`

Reply:
yea, agreed